### DISCUSSION TITLE
Mise a jour avec Click Upgrade

### DISCUSSION THREAD

**Discussion by Selleriegz** [PrestaShop/PrestaShop#36817] [Q&A]
Bonjour, je tente de mettre a jour ma boutique 1.7.6.0 vers 1.7.8 avec le module Click Upgrade, la mise à jour se lance mais s'arrête au bout de plusieurs heures j'ai donc  actualisé la page et la mise à jour a redémarrer mais le problème persiste la mise à jour s'arrête et aucun message ne s'affiche, comment faire ?

**Comment by kpodemski** [comment]
Hello @Selleriegz 

Firstly, this is an international project, and we use English in this public space. Secondly, I'm afraid you didn't provide any details. It seems best to search for someone technical to help you upgrade your PrestaShop. You could find somebody [on the official PrestaShop forum in French.](https://www.prestashop.com/forums/forum/18-forum-francophone/)

The long time to upgrade may be because you decided to backup all the files and the database, it may be because of the long-running processes that eventually timed out, etc., impossible to tell without checking.
